### PR TITLE
Add Github Actions build for Windows

### PR DESCRIPTION
**Pull request by o01eg**
It still suffers from GLEW issue but at least I could test different approaches.

### CODE REVIEW

**Comment by geoffthemedio** [comment]
I'd also add a build with `cmake .. -T v142 -A x64 -G "Visual Studio 16 2019"`

File: .github/workflows/windows.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+      - name: Build
+        run: |
+          cd build
+          cmake --build . --config RelWithDebInfo -- /verbosity:d
+      - name: Artifact
+        uses: actions/upload-artifact@v2
+        with:
+          name: Windows SDK
+          path: |
+            build/FreeOrionSDK_12_MSVC-v141-Win32.zip
```
Comment (geoffthemedio) [inline .github/workflows/windows.yml:31]: should probably call this something else, as it's not SDK version 12 anymore...

Comment (o01eg) [reply .github/workflows/windows.yml]: I could use '*' to get any version

Comment (geoffthemedio) [reply .github/workflows/windows.yml]: still not clear about when and where these get uploaded... Is it just the latest run builds that are available? for how long? where?

Comment (o01eg) [reply .github/workflows/windows.yml]: In build page in Actions tab https://github.com/o01eg/freeorion-sdk/actions/runs/2344121743

**Comment by o01eg** [comment]
`-T v143 -A x64 -G "Visual Studio 17 2022"` is available but produce error

**Comment by geoffthemedio** [comment]
I might take a look at it later, and having the test (but allowing it to fail) is probably useful

**Comment by geoffthemedio** [comment]
Is there a way to get separate .zip files to download with the artifacts instead of a single zip that contains multiple versions of the SDK? Presumably that would be more useful? Though I guess we don't make new SDKs that often and can copy them over manually, it'd probably be best if it's all automatic...? Or at least less manual?

**Comment by o01eg** [comment]
Github Releases makes separate files. It's only CI artifacts.

**Comment by geoffthemedio** [comment]
Build with the x64 version worked with my updated MSVC project files branch.
Build with the x86 failed as normal.

So this seems to work.

**Pull request merged**